keyboards/chidori/rules.mk

```diff
@@ -0,0 +1,58 @@
+SRC += matrix.c
+SRC += board.c
+SRC += i2c_master.c
+
```

Comment:
Lines 1-4 should be deleted. (We'll add them back, but modified, later in the file.)

Reply:
Modified these lines and moved to the bottom of the file.